src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingDuration.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.model.single.hearing;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class HearingDuration {
+    private Number duration;
+    private List<String> nonStandardHearingDurationReasons;
+    private Integer durationBenefitIssueCodes;
+    private Integer adjournmentDuration;
+}
```

Comment:
What is this for?

Reply:
I was following the way HearingDetailsMapping was designed, I now realise that this was not a good idea.

Reply:
Removed